config/environment.js

```diff
@@ -14,6 +14,11 @@ module.exports = function (environment) {
       },
     },
 
+    // Feature flags for the application to enable/disable certain features
+    featureFlags: {
+      detailView: false,
```

Comment:
@wolfderechter We want ALL features to always default to FALSE. Why? -> This reduces the risk that an unwanted feature will be in production as we now have to **explicitly** set it to TRUE in production
```js
  if (environment === 'production') {
     ENV.featureFlags['detailView'] = true;
  }
```